playgame.py

```diff
@@ -98,9 +121,9 @@ def parse_args():
                         action = 'store_true',
                         help = 'log moves and game state as game is played')
     parser.add_argument('-l', '--log_dir', dest = 'log_dir', default = None, 
-                        help = 'save logs to file')
+                        help = 'save logs to file (results/PATH_PROVIDED)')
```

Comment:
The request is to support any file path. If someone does "res.txt", it goes to root. "results/res.txt" makes a results directory and puts res.txt in it. Right now the help and implementation are ensuring that files go under 'results', which is not intended behavior.

I'd rather see the help file unchanged, no implicit directory, and just checking to see if a directory is parth of the path and needs to be made